scss/bitstyles/utilities/max-width/_settings.scss

```diff
@@ -0,0 +1,4 @@
+$values: (
+  '100': 100vw,
```

Comment:
i know we can override this but i'm not sure i'd assume 100 was vw and not %. so maybe the default could just be 100vw?

Reply:
Yup totally — I’ll update to be called `100vw`, and add the zero ✨